src/e2e/specs/dashboard.spec.ts

```diff
@@ -535,26 +625,6 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () =>
         "https://testrail.stage.mozaws.net/index.php?/cases/view/2463570",
     });
 
-    const clickOnATagCheckDomain = async (
```

Comment:
For future reviewers (or me in 15 minutes when i forget), this removed function was moved into utils/helpers.ts